installer/scripts/omsadmin.sh

```diff
@@ -582,6 +598,17 @@ renew_cert()
     fi
 }
 
+collectd()
+{
+    if [ -d "$COLLECTD_DIR" ]; then
+        cp $SYSCONF_DIR/omsagent.d/oms.conf $COLLECTD_DIR/oms.conf
+        cp $SYSCONF_DIR/omsagent.d/collectd.conf $CONF_DIR/omsagent.d/collectd.conf
```

Comment:
Make sure the permissions of the copied files are correct. Files in omsagent.d are owned by omsagent but files in sysconf dir are owned by root.
Note that if the omsagent agent is running when this code is executed then the plugin will only become active after a restart.